nixos/modules/services/audio/alsa.nix

```diff
@@ -21,7 +21,8 @@ in
 
       enable = mkOption {
         type = types.bool;
-        default = true;
+        default = !versionAtLeast config.system.stateVersion "18.03";
```

Comment:
Not sure about this in combination with `defaultText`, but this might cause a rebuild of the NixOS manual. Have you checked that?

Reply:
I did not check that. But if does cause a rebuild
- it doesn't really matter, as its just a single rebuild per release,
- its a bug that can surely be fixed.